apis/config/v1beta1/configuration_types.go

```diff
@@ -48,7 +48,9 @@ type Configuration struct {
 	// unsuspended, they will start immediately.
 	ManageJobsWithoutQueueName bool `json:"manageJobsWithoutQueueName"`
 
-	// ManagedJobsNamespaceSelector can be used to omit some namespaces from ManagedJobsWithoutQueueName
+	// ManagedJobsNamespaceSelector can be used to selectively exempt namespaces
+	// from management by Kueue. The selector is respected for all workload
+	// types consistently - Jobs and Pod-based integrations (pod, deployment, statefulset, etc).
 	ManagedJobsNamespaceSelector *metav1.LabelSelector `json:"managedJobsNamespaceSelector,omitempty"`
```

Comment:
Let me yet understand: 
- ManagedJobsNamespaceSelector is only relevant when `ManageJobsWithoutQueueName: true`, or regardless? Since this is replacement for the podOptions selector, it would be good to follow the same principle, but I'm not clear how the selector works if `ManageJobsWithoutQueueName: false`, and the user tries to create a Pod in kube-system with the queue-name label.

Reply:
I expanded the comment quite a bit (which then gets into the apidocs).  There is a non-regularity between Pod-based integrations and the rest of them, which at least now is documented.  Maybe there's a future KEP to tweak the semantics a bit and make things completely regular (ie, make all integrations behave like the Pod-based ones).